src/index.js

```diff
@@ -0,0 +1,188 @@
+import './styles/index.scss';
+import headerInput from './modules/header-input.js';
+import createListItem from './modules/todo-item.js';
+import createInput from './modules/editable.js';
+import addTaskToLocal from './modules/addTaskToLocal.js';
+import statusUpdate from './modules/status-update.js';
+import { storeToLocal, retrieveFromLocal } from './modules/save-retrieve.js';
+import resetIndices from './modules/reset-indices.js';
+import validateForm from './modules/formValidation.js';
+
+const todoList = document.querySelector('.todo-list');
+
+const todoListPlaceHolder = document.querySelector('.todo-list-placeholder');
+todoListPlaceHolder.prepend(headerInput());
+
+const populateList = (list) => {
+  list.forEach((item) => {
+    const { description, id, completed } = item;
+    const listItem = createListItem(id, description);
+    if (completed) {
+      listItem.querySelector(`#task${id}`).classList.add('active');
+      listItem.children[0].checked = true;
+    }
+    todoList.appendChild(listItem);
+  });
+};
```

Comment:
I like the way you targetted the checkbox by using its position as the first child of the parent container 🤟 👍 👏 . It's impressive.